packages/web/public/static/locales/de/comment.json

```diff
@@ -9,5 +9,8 @@
   "tabGeneralKey": "Allgemein",
   "tabOutdatedKey": "Veraltet",
   "noOutdatedThreadsMessage": "Es gibt keine veralteten Threads",
```

Comment:
Hi @LindasLinguas ! There are a few new strings below to be translated, if you could help us with those it would be amazing! No rush, I think we still have at least 2 weeks of work to complete this 😊